frontend/js/src/user/stats/components/UserGenreDayActivity.tsx

```diff
@@ -240,6 +236,101 @@ export default function UserGenreDayActivity({
     });
   }, [colorScale, rawData, timezoneOffset]);
 
+  // Responsive time markers - all four markers with mobile-optimized positioning
+  const getTimeMarkersConfig = () => {
+    if (isMobile) {
+      return [
+        {
+          position: {
+            top: "15px",
```

Comment:
While playing with the markers, I came up with something that seems to work for all sizes, with the markers inside the circle.
I'm not sure it's better than the current implementation as it changes the overall look, but seems to be more flexible as the placement does not change depending on the parent container size:
<img width="714" height="629" alt="image" src="https://github.com/user-attachments/assets/c14bbaf6-1ced-4e20-afaa-2c165bcb4088" />

`top: "calc(50% - 80px)"`  <- this calculated position works the same for every marker, but applied to the correct attribute ('top' for the top one, 'bottom' for the bottom one, etc.)
For the smaller screen sizes the 80px needs to be 40px (the same value as the right and left margins defined in getChartConfig)